src/theme/components/Footer.tsx

```diff
@@ -10,7 +10,7 @@ import { helmet } from '../../content/HelmetContent';
 
 const Footer = (): JSX.Element => {
   const { title, socialLinks } = landing;
-  const { profile } = helmet;
+  const { logo } = helmet;
```

Comment:
⚠️ The footer now displays the image created for og:image
We should restore the usage of the white logo that is also used in the header.

![image](https://user-images.githubusercontent.com/27200110/130647995-2adb9f63-af96-42da-a234-1b594c8f342a.png)

Reply:
Fixed